src/System.Runtime.Extensions/ref/System.Runtime.Extensions.csproj

```diff
@@ -14,5 +14,8 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../System.Runtime/ref/System.Runtime.csproj" />
```

Comment:
Can we add a ToDo here as well to track the work of removing this dependency and use the System.Runtime package once it's pushed?